alfresco-transform-libreoffice/alfresco-transform-libreoffice/src/main/resources/libreoffice_engine_config.json

```diff
@@ -197,39 +197,39 @@
         {"sourceMediaType": "application/vnd.ms-excel",                                                                                                  "targetMediaType": "application/vnd.oasis.opendocument.spreadsheet-template" },
         {"sourceMediaType": "application/vnd.ms-excel",                                                                                                  "targetMediaType": "text/csv" },
         {"sourceMediaType": "application/vnd.ms-excel",                                                                                                  "targetMediaType": "text/tab-separated-values" },
-        {"sourceMediaType": "application/vnd.ms-excel",                                                  "maxSourceSizeBytes": 10485760,                 "targetMediaType": "application/pdf" },
+        {"sourceMediaType": "application/vnd.ms-excel",                                                  "maxSourceSizeBytes": 12582912,                 "targetMediaType": "application/pdf" },
```

Comment:
btw this one is 18874368 in T-Service

Reply:
Great catch - it's been a long few days. Fixing now.

Reply:
Fixed.